docs/ui/components/Sidebar/SidebarHeadEntry.tsx

```diff
@@ -14,14 +14,14 @@ type SidebarHeadEntryProps = {
 export const SidebarHeadEntry = ({ href, title, isActive, Icon }: SidebarHeadEntryProps) => {
   return (
     <Link href={href} passHref>
-      <div css={[entryContainerStyle, isActive && activeEntryContainerStyle]}>
+      <a css={[entryContainerStyle, isActive && activeEntryContainerStyle]}>
```

Comment:
This is lacking `aria-current="page"` if it's currently selected. It changes the announcing to:

```
Guides visited link current page
```